packages/lexical-selection/src/lexical-node.ts

```diff
@@ -243,16 +243,18 @@ function $patchStyle(
   target: TextNode | RangeSelection,
   patch: Record<
     string,
-    string | null | ((currentStyleValue: string | null) => string)
+    | string
+    | null
+    | ((currentStyleValue: string | null, _target: typeof target) => string)
```

Comment:
Is there a need to underscore this parameter name? Does it get caught by the eslint shadow rule?

```suggestion
    | ((currentStyleValue: string | null, target: typeof target) => string)
```

Reply:
Yes, shadow rule complains.

<img width="840" alt="Screenshot 2024-08-01 at 5 24 59 PM" src="https://github.com/user-attachments/assets/4db66133-c39d-47af-ba26-7b72d397da4b">